pkgs/development/python-modules/sphinx/default.nix

```diff
@@ -64,7 +64,7 @@ buildPythonPackage rec {
     alabaster
     docutils
     imagesize
-    jinja2
+    (jinja2.override { enableDocumentation = false; })
```

Comment:
same, probably needs a comment alongside.

Reply:
To avoid infinity loops. jinja2 docs depends on sphinx, sphinx depends on jinja, so you can't build sphinx with jinja that includes documentation.